server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/exchanges/dvchain/DVChainExchange.java

```diff
@@ -82,8 +95,37 @@ public BigDecimal getCryptoBalance(String cryptoCurrency){
         return BigDecimal.ZERO;
     }
 
+    @Override
+    public BigDecimal getFiatBalance(String fiatCurrency){
+        return BigDecimal.valueOf(10000000);
```

Comment:
Add a check of fiatCurrency parameter against getFiatCurrencies(). If it's not there, return null.

Reply:
Will do